packages/cli/src/index.ts

```diff
@@ -14,6 +25,27 @@ program
     initCommand();
   });
 
+program
+  .command('sync')
+  .option(
+    '-n, --novu-cloud-api-url <novuCloudApiUrl>',
+    'The Novu Cloud API url, defaults to https://api.novu.co',
+    'https://api.novu.co'
+  )
+  .requiredOption('--novu-endpoint-url <novuEndpointUrl>', 'The novu endpoint URL')
+  .requiredOption('--api-key <apiKey>', 'The Novu api key')
+  .description('Sync your workflow state with Novu Cloud')
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** ‏ adjusted description, let me know what you think

Reply:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** It's only workflows right now, but will expand to other domains soon. The description should reflect this direction.
‏
```suggestion
  .description('Sync your Novu Framework state with Novu Cloud')
```